ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+"""
+fpkm_scatter.py
+
+Takes FPKM counts from two conditions and makes a scatter plot.
+Also calculates a r-squared correlation score.
+
+Either takes two Cufflinks FPKM summary files or a summary file
+with multiple samples.
+"""
+
+from __future__ import print_function
```

Comment:
The print function is vastly superior in every way to the statement and I will gladly armwrestle or thumbwar anyone who puports otherwise. It lets you do all kinda neat things but the one I use most often is `print("Some text", file=sys.stderr)`